docs/cmdlet-example/README.md

```diff
@@ -0,0 +1,57 @@
+# Building a C# Cmdlet
+
+This demonstrates how to build your own C# cmdlet for PowerShell Core with Visual Studio.
+
+We will use free Visual Studio Community 2017 that can be downloaded from [https://www.visualstudio.com/downloads](https://www.visualstudio.com/downloads)
```

Comment:
You can just make 'Visual Studio Community 2017' the link instead of making the hyperlink text the hyperlink

Reply:
Updated.